### ISSUE TITLE
AccessibleSlider doesn't work with mobile VoiceOver

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/sun#397]
Found while working on https://github.com/phetsims/scenery/issues/852, AccessibleSlider doesn't work with gesture input on mobile VoiceOver.

It isn't receiving any input events when flicking up and down on the screen when VO is enabled. I discovered that when this line is removed, the problem goes away:
```
// value 'any' allows input to have values that are not evenly divisible by the step size, which is
// required for PhET sliders, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-step
this.setAccessibleAttribute( 'step', 'any' );
```

**Comment by jessegreenberg** [comment]
From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-step

> If this attribute is not set to any, the control accepts only values at multiples of the step value greater than the minimum.

Ill see if `step="any"` works in mobile safari/VO outside of a PhET sim. If it does, we can troubleshoot this use case. If not, maybe there is another way to support range inputs with values that are not evenly divisible by the step size.

**Comment by jessegreenberg** [comment]
Well... if step=any, how would the browser know how much to change the value by from a gesture?

**Comment by jessegreenberg** [comment]
Working with the above commits. I removed the `any` value for the `step` attribute, instead just using a value that evenly divides the range. Also, there was an issue in scenery where the `value` for the primary sibling wasn't getting updated correctly, that was fixed as well. Closing.
